ambry-replication/src/main/java/com.github.ambry.replication/ReplicaThread.java

```diff
@@ -465,7 +465,9 @@ private void processReplicaMetadataResponse(Set<StoreKey> missingStoreKeys,
           MessageFormatInputStream deleteStream =
               new DeleteMessageFormatInputStream(messageInfo.getStoreKey(), messageInfo.getAccountId(),
                   messageInfo.getContainerId(), messageInfo.getOperationTimeMs());
-          MessageInfo info = new MessageInfo(messageInfo.getStoreKey(), deleteStream.getSize(), true);
+          MessageInfo info =
```

Comment:
deleteMsgStream size might differ depending on what version current node serialized the stream and hence can't rely on mesageInfo.